tests/main/interfaces-mount-control/task.yaml

```diff
@@ -4,7 +4,9 @@ environment:
     MOUNT_SRC: /var/tmp/test-snapd-mount-control
     SNAP_COMMON: /var/snap/test-snapd-mount-control/common
     SNAP_NAME: test-snapd-mount-control
-    MOUNT_DEST: $SNAP_COMMON/target
+    MOUNT_DEST1: $SNAP_COMMON/target1
+    MOUNT_DEST2: $SNAP_COMMON/target2/path
+    MOUNT_DEST3: $SNAP_COMMON/target3/path/subpath
```

Comment:
maybe this should have each a comment before with a summary of the entries in the plug?

```
# what: /usr/** where: $SNAP_COMMON/target3/**
MOUNT_DEST3: $SNAP_COMMON/target3/path/subpath
```